python/ray/rllib/a3c/a3c.py

```diff
@@ -26,17 +26,15 @@ class Runner(object):
 
     The gradient computation is also executed from this object.
     """
-    def __init__(self, env_name, actor_id, logdir, start=True):
+    def __init__(self, env_name, policy_cls, actor_id, batch_size, logdir):
         env = create_env(env_name)
         self.id = actor_id
-        num_actions = env.action_space.n
-        self.policy = LSTMPolicy(env.observation_space.shape, num_actions,
-                                 actor_id)
-        self.runner = RunnerThread(env, self.policy, 20)
+        # Todo: should change this to be just env.observation_space
```

Comment:
Can you make this `TODO(rliaw):` so that people know who to contact if they have questions :)